tests/comfy_cli/cql/test_dynamic_combo_link_subs.py

```
@@ -0,0 +1,172 @@
+        )
+        wf["last_node_id"] = 4
+        before = list(node["widgets_values"])
+        wf, _op = workflow_ops.set_widget(wf, graph, 4, "seed", 4242)
+        after = wf["nodes"][0]["widgets_values"]
+        assert len(after) == len(before)
+        assert after[5] == 4242  # the frontend's seed position
+        assert after[:5] == before[:5] and after[6:] == before[6:]
+
+
+class TestAddNodeMatchesTheCanvas:
+    def test_add_node_writes_the_frontend_slot_count(self, graph: Graph, frontend_nodes: dict):
```

Pylint (4.0.7)

[convention] Missing function or method docstring

(C0116)